documentation/source/building.rst

```diff
@@ -71,9 +74,10 @@ Cocotb
     Use this to override the default behavior of annotating cocotb output with
     ANSI color codes if the output is a terminal (``isatty()``).
 
-    ``COCOTB_ANSI_OUTPUT=1`` forces output to be ANSI regardless of the type of ``stdout`` or the presence of :envvar:`NO_COLOR`.
-
-    ``COCOTB_ANSI_OUTPUT=0`` suppresses the ANSI output in the log messages
+    ``COCOTB_ANSI_OUTPUT=1``
+       forces output to be ANSI regardless of the type of ``stdout`` or the presence of :envvar:`NO_COLOR`
+    ``COCOTB_ANSI_OUTPUT=0``
+       suppresses the ANSI output in the log messages
```

Comment:
ANSI what? ANSI is an organization. Should be corrected to "ANSI colors" or "ANSI color codes"

Reply:
Old text, but yeah.

Reply:
I understand, good to fix it now before we forget.